cedar-policy-core/src/est/expr.rs

```diff
@@ -346,11 +340,6 @@ impl Expr {
         Expr::ExprNoExt(ExprNoExt::Slot(slot))
     }
 
-    /// Partial-evaluation unknowns
-    pub fn unknown(name: impl Into<SmolStr>) -> Self {
-        Expr::ExprNoExt(ExprNoExt::Unknown { name: name.into() })
-    }
```

Comment:
If it would be convenient, it would be possible to keep this constructor and just change its implementation to produce the new representation of `unknown`